playwright.config.ts

```
@@ -55,9 +55,11 @@ export default defineConfig({
     // },
   ],
 
-  webServer: {
-    command: 'pnpm e2e:dev',
-    port,
-    reuseExistingServer: !process.env.CI,
-  },
+  webServer: process.env.PLAYWRIGHT_BASE_URL
```

ESLint

[error] Unexpected use of the global variable 'process'. Use 'require("process")' instead.

(node/prefer-global/process)